### ISSUE TITLE
Need support for single quote expression

### ISSUE THREAD

**Issue by andrew-qiu** [vincentrussell/json-data-generator#17]
I found the recommended placeholder style, like this: 

```json 
{
    "day1": "{{put("date", date("dd-MM-yyyy HH:mm:ss"))}}",
    "day2": "{{addDays(get("date"), 12)}}"
}
```

was hard to convert to a human friendly format with the most common tools. So I expect json-data-generator could support the single-quote or non-quote placeholder expression, like this:

```json 
{
    "day1": "{{put('date', date('dd-MM-yyyy HH:mm:ss'))}}",
    "day2": "{{addDays(get('date'), 12)}}"
}
```

**Comment by vincentrussell** [comment]
Sorry.  Why wont the double quotes work for you?

**Comment by andrew-qiu** [comment]
> Sorry. Why wont the double quotes work for you?

When I tried to use some common json formatter tools, for example, https://jsoneditoronline.org/. They did not work with the following text:

```json
{"day1": "{{put("date", date("dd-MM-yyyy HH:mm:ss"))}}","day2":"{{addDays(get("date"), 12)}}"}
```

but work well with the single-quote format text:

```json
{"day1": "{{put('date', date('dd-MM-yyyy HH:mm:ss'))}}","day2": "{{addDays(get('date'), 12)}}"}
```

**Comment by vincentrussell** [comment]
I'll take a look when I get a chance in a few days.

**Comment by vincentrussell** [comment]
So my experience was the opposite of yours.  Double-quoted strings worked for me, but single quoted strings didn't.     I added support for single-quoted strings.  See the following commit:

e302b5243d885e51088a3b338ea64e948c7f0387

**Comment by vincentrussell** [comment]
I just released version 1.8

**Comment by andrew-qiu** [comment]
> I just released version 1.8

Thanks, Russell
